sklearn/ensemble/voting_classifier.py

```diff
@@ -47,6 +58,10 @@ class VotingClassifier(BaseEstimator, ClassifierMixin, TransformerMixin):
         predicted class labels (`hard` voting) or class probabilities
         before averaging (`soft` voting). Uses uniform weights if `None`.
 
+    n_jobs : int, optional (default=1)
+        The number of jobs to run in parallel for both `fit`.
```

Comment:
"both" should be removed.

fit won't show as code/tt unless you put it in double-backticks.